concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1694,7 +1694,11 @@ jobs:
                 fi
 
                 # TODO: remove '-o admin' when we are happy to release this to the world
-                cf enable-service-access aws-s3-bucket -o admin
+                if cf service-access | grep "aws-s3-bucket\s"; then
+                  echo "already enabled"
+                else
+                  cf enable-service-access aws-s3-bucket -o admin
+                fi
```

Comment:
Do we need to do this at all?  I'm not clear why we need this enabled in the admin org...

Reply:
true it could come out completely.

Reply:
I already fixed this in a different way, where did that go? o_O

Reply:
It was accidentally overridden with a force push :(

https://github.com/alphagov/paas-cf/compare/5e559ee5c099f760bff381ccd9e4b73f5d69dba5..c5863b7864bf68d94059c6826d51f34565848c13

Reply:
@bandesz I've switched this out for your method as I prefer it.